frontend/src/LayersSelector.tsx

```diff
@@ -86,7 +88,16 @@ export const LayersSelector = (props: {
     <input
       type='checkbox'
       checked={state.windLayerEnabled}
-      onChange={() => enableWindLayer(!state.windLayerEnabled)}
+      onChange={() => props.domain.enableWindLayer(!state.windLayerEnabled)}
+    />
+  );
+  const windNumericValuesCheckBox = inputWithLabel(
+    'Numerical values',
+    'Show numerical values instead of barbells',
```

Comment:
Not sur what a barbell is, how about "wind strength arrows" or "wind barbs"?

Reply:
Good point, I badly translated “barbule”… I will fix that.

Reply:
Done in ea42b92.